modules/doc/content/python/MooseDocs/extensions/gitutils.md

```diff
@@ -0,0 +1,27 @@
+# GitUtils Extension
+
+The "gitutils" extension provides a mechanism for gleaning information regarding the git repository
+that the documentation build command is being executed within.
+
+!devel settings module=MooseDocs.extensions.gitutils
+                object=GitUtilsExtension
+                id=gitutils-config
+                caption=Configuration items for the "gitutils" extension.
+
+## Commit
+
+The "commit" sub-command inserts the current commit SHA-1 for the repository that the documentation
+build occurs as demonstrated in [gitutils-commit-example].
```

Comment:
"occurs in as demonstrated" or "for the repository containing the documentation as demonstrated"